apps/src/templates/VideoThumbnail.jsx

```diff
@@ -6,30 +6,35 @@ import {videoDataShape} from './types';
 export default class VideoThumbnail extends Component {
   static propTypes = {
     video: videoDataShape,
-    onClick: PropTypes.func
+    onClick: PropTypes.func,
+    openInNewTab: PropTypes.bool
+  };
+
+  onThumbnailClick = () => {
```

Comment:
add a test?

Reply:
ha, turns out neither of these files ever had any tests 🙃 I'll add a couple of basic ones for this file and `ResourceLink`